core/dbt/contracts/graph/parsed.py

```diff
@@ -202,6 +203,14 @@ def node_info(self):
         node_info_msg = NodeInfo(**node_info)
         return node_info_msg
 
+    def update_event_status(self, **kwargs):
```

Comment:
Should we consider naming and typing the 'valid' parameters to this function?

Reply:
Yeah, we could do that. Would make the code to set them more complicated. I'm going to leave for future enhancement.